platforms/ios/README.md

```diff
@@ -0,0 +1,63 @@
+iOS
+===
+
+The recommended way to use tangram-es in an iOS project is to add it as a CocoaPods dependency. The library is hosted in CocoaPods under the pod name 'Tangram-es'. To find the latest version, check CocoaPods: https://cocoapods.org/pods/Tangram-es.
+
+## Setup ##
+
+This project uses CMake (minimum version 3.0), you can download it [here](http://www.cmake.org/download/) or use your favorite installation package tool like [homebrew](http://brew.sh/).
+
+```bash
+brew install cmake
+```
+
+Make sure to update git submodules before you build:
+
+```bash
+git submodule update --init
```

Comment:
We still need recursive? (I might have missed some pr in last couple of days from Hannes or karim which removes recursive submodules)